README.md

```diff
@@ -1,7 +1,17 @@
-![build status](https://travis-ci.org/jmdns/jmdns.svg)
-[![Maven Central](https://maven-badges.herokuapp.com/maven-central/org.jmdns/jmdns/badge.svg)](https://maven-badges.herokuapp.com/maven-central/org.jmdns/jmdns)
-[![Javadocs](http://www.javadoc.io/badge/org.jmdns/jmdns.svg)](http://www.javadoc.io/doc/org.jmdns/jmdns)
+<div align="center">
+<img src="https://github.com/quarkiverse/quarkus-mdns/blob/main/docs/modules/ROOT/assets/images/mdns.png" height="100" >
```

Comment:
I was already thinking about creating a logo for JmDNS - it seems the Quarkus guys did that already. :-)
We still shouldn't simply reference their image. I would suggest to:
1. Open an issue at quarkiverse/quarkus-mdns, asking them if they are fine that we adopt the logo they have created.
2. Commit the logo in our own repo and reference it from there.